cli/parse_pdfs.py

```diff
@@ -277,9 +279,39 @@ def parse_file(
                 },
             )
             return None
+        except HttpResponseError as e:
+            _LOGGER.error(
+                "Failed to parse document with Azure API with default endpoint, "
+                "retrying with large document endpoint.",
+                extra={
+                    "props": {
+                        "document_id": input_task.document_id,
+                        "error_message": str(e.message),
+                    }
+                },
+            )
+            try:
+                response_: Tuple[
+                    Sequence[PDFPage], AnalyzeResult
```

Comment:
Is it correct that we're just throwing away the `PDFPage` objects?

Reply:
The `Sequence[PDFPage]` is all the api responses for the individual pages. This might be useful in the future but currently isn't required.